src/style.css

```diff
@@ -1,3 +1,66 @@
 body {
+  font-family: Roboto, sans-serif;
+  font-weight: 500;
   margin: 0;
 }
+
+header {
+  display: flex;
+  justify-content: space-between;
+  padding: 0 50px;
+  align-items: center;
+}
```

Comment:
1. it's a good practice to write all styles using classes -> add class to tag header 
2. add height here